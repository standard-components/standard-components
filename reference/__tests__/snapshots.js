import snapshot from '@compositor/kit-snapshot'
import * as examples from '../examples'

snapshot(examples)
