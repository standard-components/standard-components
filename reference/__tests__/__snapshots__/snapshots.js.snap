// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundImage renders 1`] = `null`;

exports[`Box renders 1`] = `null`;

exports[`BoxColor renders 1`] = `null`;

exports[`Button renders 1`] = `null`;

exports[`Flex renders 1`] = `null`;

exports[`Heading renders 1`] = `null`;

exports[`Image renders 1`] = `null`;

exports[`Link renders 1`] = `null`;

exports[`Text renders 1`] = `null`;
